snap/remote/remote.go

```diff
@@ -47,4 +46,8 @@ type Snap struct {
 	Title           string             `json:"title"`
 	Type            snap.Type          `json:"content,omitempty"`
 	Version         string             `json:"version"`
+
+	// FIXME: the store should return "developer" to us instead of
+	//        origin
+	Developer string `json:"origin" yaml:"origin"`
```

Comment:
Is the "yaml" here necessary?  Do we use this type to sync from disk or anywhere else besides the store?

One alternative would be to keep the Origin field unchanged, and introduce the Developer field that is manually populated on queries from the store. Best approach depends on how we use this.